guides/Pagination.md

```diff
@@ -1,4 +1,68 @@
-# Pagination
+# Sorting and Paging
+
+_List_ and _Query_ endpoints that return a list of entities
+allow you to specify sorting and paging options in the request.
+
+This articles gives a general overview of sorting and paging.
+The implementation changes
+depending on whether an endpoint is a GET or POST request.
+Check your API's documentation for specific details,
+including which fields are sortable.
+
+## Sorting
+
+Most APIs default to sorting by `createdDate` in `DESC` (descending) order,
+although some APIs have a different default sort order.
+You can override the default sorting by specifying
+a new field and order.
+
+### Sort _List_ endpoints
+
+_List_ endpoints are designed to be lightweight HTTP GET requests.
+For this reason, sorting is applied through the query parameters,
```

Comment:
This is a conundrum. I KNOW these are called query parameters in the URL for the call of the endpoint. But it's unfortunate that it sounds initially to the ear as misleading, looks like a mistake, because we are talking about list endpoints. I don't know that there is anything to do here, but I did do an initial double-take when I read this. @imnotashrimp